flink-filesystems/flink-gs-fs-hadoop/src/test/java/org/apache/flink/fs/gs/writer/GSRecoverableFsDataOutputStreamTest.java

```diff
@@ -133,57 +131,57 @@ public void before() {
         }
     }
 
-    @Test
+    @TestTemplate
     public void emptyStreamShouldHaveProperPositionAndComponentObjectCount() {
         if (empty) {
-            assertEquals(0, position);
-            assertEquals(0, componentObjectCount);
+            assertThat(position).isZero();
+            assertThat(componentObjectCount).isOne();
```

Comment:
This should be `.isZero()`, and causes the Azure fail. Missed that in my prev comment. Pls. fix this, if tests are green, then LGTM :)